app/imports/ui/pages/AddEvents.jsx

```diff
@@ -1,11 +1,95 @@
 import React from 'react';
-import { Container } from 'react-bootstrap';
-import HelpButton from '../components/HelpButton';
+import { Card, Col, Container, Row } from 'react-bootstrap';
+import { AutoForm, ErrorsField, TextField, LongTextField, SubmitField, SelectField } from 'uniforms-bootstrap5';
+import swal from 'sweetalert';
+import { Meteor } from 'meteor/meteor';
+import SimpleSchema2Bridge from 'uniforms-bridge-simple-schema-2';
+import SimpleSchema from 'simpl-schema';
+import { Events } from '../../api/event/Event';
+import { Companies } from '../../api/company/Company';
+// Create a schema to specify the structure of the data to appear in the form.
+const formSchema = new SimpleSchema({
+  eventName: String,
+  image: String,
+  address: String,
+  description: String,
+  tag: {
+    type: String,
+    allowedValues: ['Computer Science', 'Cyber security', 'Web Development', 'Data Science', 'Business Administration',
+      'Marketing', 'Accounting', 'Finance', 'Entrepreneurship', 'Biology', 'Chemistry',
+      'Physics', 'Environmental Science', 'Geology', 'Psychology', 'Sociology', 'Political Science',
+      'Economics', 'Anthropology', 'English', 'History', 'Philosophy', 'Religious Studies', 'Classics',
+      'Fine Arts', 'Music', 'Theater', 'Film', 'Creative Writing', 'Nursing', 'Pre-Med', 'Public Health',
+      'Health Sciences', 'Physical Therapy', 'Linguistics', 'Journalism', 'Advertising', 'Public Relations',
+      'Communication Studies', 'Law', 'Criminal Justice', 'Paralegal Studies', 'Political Science', 'Sociology',
+      'International Relations', 'Global Studies', 'Foreign Languages', 'Engineering'].sort(),
+    defaultValue: 'Accounting',
+  },
+  companyId: {
+    type: String,
+    defaultValue: Companies.companyPublicationName,
+  },
+  createdAt: {
+    type: Date,
+    defaultValue: new Date(),
+  },
+});
 
-const AddEvents = () => (
-  <Container className="py-3">
-    <HelpButton />
-  </Container>
-);
+const bridge = new SimpleSchema2Bridge(formSchema);
 
-export default AddEvents;
```

Comment:
Renaming from AddEvents to AddEvent is fine, but we should make it consistent and rename the file as well